README.md

```diff
@@ -1 +1,308 @@
 # backend-capstone-template
+# ChatGPT Devs Era
+
+
+ChatGPT Devs Era is a website, customers can browse through various cooks and their menus, discovering a wide range of homemade food options. They can easily place orders for the meals they desire and have them delivered or arrange for pick-up.
+For Cooks, the platform provides a simple way to create profiles and showcase their homemade dishes. They can manage their menu, track orders, and interact with their customers.
+
+## Table of Contents
+
+- [Purpose](#purpose)
+- [Features](#features)
+- [Installation](#installation)
+- [Usage](#usage)
+- [Examples](#examples)
+- [Contributing](#contributing)
+- [License](#license)
+
+
+## Purpose
+
+1.	Connect customers with cooks who offer homemade food.
+2.	Support local cookers and help customers who are looking for homemade food
+
+
+
+## Features
+
+1.	Connects customers with local cooks offering homemade food.
+2.	Browse and explore a diverse range of homemade dishes.
+3.	User-friendly interface for placing and managing orders.
+4.	Profiles for cooks to showcase their culinary offerings.
+5.	Order tracking and communication between customers and cooks.
+
+## Installation
+
+To install ChatGPT Devs Era, you can use yarn:
+
+```bash
+yarn install chatgpt-devs-era
+```
+
+### To start the server
+
+run the command `yarn && yarn start`. This will install all the dependencies
+
```

Comment:
This is not accurate, so if we do
- just `yarn` it will install the needed dependencies
- `yarn & yarn start` it will install the dependencies and start the server
-  just `yarn start` will start the server and this is what we need

So instead of saying  `run the command `yarn && yarn start`. This will install all the dependencies
We should say `run the command `yarn start`. This will install all the dependencies